astrogrism/config/common/WFSS_Wavelength_Range_V1.1_with_ACS.tab

```diff
@@ -0,0 +1,178 @@
+JWST NIRCAM C modB F430M +1 4.121000000000004 4.448500000000004
+JWST NIRCAM C modB F430M +2 4.201000000000004 4.389000000000004
+JWST NIRCAM R modA F300M +1 2.7610000000000023 3.259000000000002
+JWST NIRCAM R modA F300M +2 2.791000000000002 3.179000000000002
+JWST NIRCAM C modB F322W2 +1 2.391000000000002 4.139000000000001
+JWST NIRCAM C modB F322W2 +2 2.381000000000002 3.5990000000000015
+JWST NIRCAM C modB F277W +1 2.3710000000000018 3.229000000000002
+JWST NIRCAM C modB F277W +2 2.3710000000000018 3.149000000000002
+JWST NIRISS GR150C "" F150W +1 1.288 1.712
+JWST NIRISS GR150C "" F150W 0 1.288 1.712
+JWST NIRISS GR150C "" F150W +2 1.3387046941544807 1.7143780419540788
+JWST NIRISS GR150C "" F150W +3 1.3493257021002125 1.7143780419540788
+JWST NIRISS GR150C "" F150W -1 1.3493257021002125 3.0510001953125
+JWST NIRISS GR150R "" F090W +1 0.7887001774306214 1.031
+JWST NIRISS GR150R "" F090W 0 0.7887001774306214 1.031
+JWST NIRISS GR150R "" F090W +2 0.7865217488098764 1.0265246003059216
+JWST NIRISS GR150R "" F090W +3 0.787 1.029
+JWST NIRISS GR150R "" F090W -1 0.787454623338691 3.396
+JWST NIRISS GR150R "" F200W +1 1.7029009582509522 2.279451922495031
+JWST NIRISS GR150R "" F200W 0 1.7029009582509522 2.279451922495031
+JWST NIRISS GR150R "" F200W +2 1.7096428456142785 2.280352767465269
+JWST NIRISS GR150R "" F200W +3 1.7103185006448478 2.280352767465269
+JWST NIRISS GR150R "" F200W -1 1.713 2.2810001953125
+JWST NIRISS GR150R "" F115W +1 0.9973383916486066 1.3187545575193975
+JWST NIRISS GR150R "" F115W 0 0.9973383916486066 1.3187545575193975
+JWST NIRISS GR150R "" F115W +2 0.9926207124409168 1.2919352743310675
+JWST NIRISS GR150R "" F115W +3 0.993 1.2934676076862788
+JWST NIRISS GR150R "" F115W -1 0.9937980359933947 3.092
+JWST NIRISS GR150R "" F150W +1 1.288 1.712
+JWST NIRISS GR150R "" F150W 0 1.288 1.712
+JWST NIRISS GR150R "" F150W +2 1.3387046941544807 1.7143780419540788
+JWST NIRISS GR150R "" F150W +3 1.3493257021002125 1.7143780419540788
+JWST NIRISS GR150R "" F150W -1 1.3493257021002125 3.0510001953125
+JWST NIRISS GR150C "" F090W +1 0.7887001774306214 1.031
+JWST NIRISS GR150C "" F090W 0 0.7887001774306214 1.031
+JWST NIRISS GR150C "" F090W +2 0.7865217488098764 1.0265246003059216
+JWST NIRISS GR150C "" F090W +3 0.787 1.029
+JWST NIRISS GR150C "" F090W -1 0.787454623338691 3.396
+JWST NIRISS GR150C "" F200W +1 1.7029009582509522 2.279451922495031
+JWST NIRISS GR150C "" F200W 0 1.7029009582509522 2.279451922495031
+JWST NIRISS GR150C "" F200W +2 1.7096428456142785 2.280352767465269
+JWST NIRISS GR150C "" F200W +3 1.7103185006448478 2.280352767465269
+JWST NIRISS GR150C "" F200W -1 1.713 2.2810001953125
+JWST NIRISS GR150C "" F115W +1 0.9973383916486066 1.3187545575193975
+JWST NIRISS GR150C "" F115W 0 0.9973383916486066 1.3187545575193975
+JWST NIRISS GR150C "" F115W +2 0.9926207124409168 1.2919352743310675
+JWST NIRISS GR150C "" F115W +3 0.993 1.2934676076862788
+JWST NIRISS GR150C "" F115W -1 0.9937980359933947 3.092
```

Comment:
I'm all for eventually expanding astrogrism beyond HST, and I think including these values puts us in a good spot to expand our work. I do wonder whether we should have them here though if we don't do anything with JWST yet?

Reply:
I figured it's better to have them then remove them and have to track them down again later. Doesn't hurt anything to have them here, this isn't a file users should ever be looking at 🤷